src/fiber/fiber.ml

```diff
@@ -459,7 +459,14 @@ module Var = struct
   let create () = create ~name:"var" (fun _ -> Dyn.Encoder.string "var")
 end
 
-let with_error_handler f ~on_error k = EC.set_error_handler ~on_error f () k
+let with_error_handler_internal f ~on_error k =
+  EC.set_error_handler ~on_error:(fun x -> on_error x) f () k
```

Comment:
```suggestion
  EC.set_error_handler ~on_error f () k
```

Reply:
I don't know what you're talking about, pretty sure there was never any change. :-)